actors/builtin/miner/miner_actor.go

```diff
@@ -652,17 +649,20 @@ func (a *Actor) checkPrecommitExpiry(rt Runtime, sectorNo abi.SectorNumber) {
 		return
 	}
 
-	if rt.CurrEpoch()-sector.PreCommitEpoch > PoRepMaxDelay {
-		rt.State().Transaction(&st, func() interface{} {
-			err = st.deletePrecommitttedSector(store, sectorNo)
-			if err != nil {
-				rt.Abort(exitcode.ErrIllegalState, "failed to delete precommit %v: %v", sectorNo, err)
-			}
-			return nil
-		})
-		_, code := rt.Send(builtin.BurntFundsActorAddr, builtin.MethodSend, nil, sector.PreCommitDeposit)
-		builtin.RequireSuccess(rt, code, "failed to burn funds")
+	if rt.CurrEpoch()-sector.PreCommitEpoch <= MaxSealDuration[regProof] {
+		rt.Abort(exitcode.ErrIllegalState, "Precommit not expired %v: %v", sectorNo, err)
```

Comment:
Aborting here isn't particularly helpful. It will merely abort the call from cron and prevent further cleanup progress.

Reply:
Gotcha. Reverted.